Makefile

```diff
@@ -159,3 +159,8 @@ docs-build:
 	mdbook build
 	rsync --recursive --delete-after --exclude 'v4.*' --exclude .git\
 		./book/ $(DOCS_DIR)/
+
+.PHONY: unit-test
+unit-test: vendor
```

Comment:
small nit, but would you mind making this similar to the makefile in claircore. 

https://github.com/quay/claircore/blob/main/Makefile#L29

just for sanity sake, using the same keyword is helpful.

Reply:
Sure, I'll just match them and the coverage stuff can be a discussion outside of this.

Reply:
done